docs/fieldnorm/life_cycle.md

```diff
@@ -1,7 +1,56 @@
 # Software life cycle processes
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris at bibendum metus. Etiam arcu augue, aliquet vel convallis eu, finibus ac arcu. Morbi ac tellus condimentum, pretium justo at, facilisis neque. Integer vulputate, tortor in ultricies bibendum, magna justo egestas nibh, a gravida tellus orci quis mi. Nam dictum orci et arcu tincidunt condimentum. Nam ac mauris arcu. Pellentesque sagittis turpis sed bibendum interdum. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Aenean bibendum pretium lacinia. Donec posuere, tellus facilisis iaculis ornare, nisl felis luctus urna, ac lacinia velit enim non enim.
+One of the main responsibilities of Bioinformaticians within a diagnostic setting involves the development and maintenance of software used in diagnostic processes. Because ISO-15189 does not cover software development, it is essential that inhouse software is created and maintained in accordance with recognized industry standards, such as IEC-62304 and IVDR appendix I. The IEC-62304 description of the software development life cycle (SDLC) is consistent with the practices in most software-centric entities. SDLC processes that are commonly used include planning, requirements, architectural design, development, testing, deployment and maintenance. Bioinformatics software development within a ISO-15189 certified genetics laboratory should establish clear guidelines that cover all SDLC stages. 
```

Comment:
change "industry standards" to "harmonized standards" 
IVDR is not a standard; we need to add IVDR to a general Introduction of the Fieldnorm I think

Reply:
Maybe add the first few sentences to a general introducion. And start with  "SDLC"

Reply:
"should  document the different stages of DLC"

Reply:
Do we want "should document" or "should have procedure". Taking into account the clear distinction the ISO makes on things that have to be documented and things that should have "procedures". This is something we as a team should decide if we want clear documentation or more freedom.